commands/test.js

```diff
@@ -64,7 +64,9 @@ exports.run = async function({dir = '.', cover, watch, skipBuild}) {
   await runTests();
 
   if (watch) {
-    compiler.on('done', runTests);
+    compiler.on('done', () => {
+      runTests().catch(() => {}); // ignore error, since test output is already actionable enough
```

Comment:
Should we log something here?

Reply:
I think logging would make the test output unnecessarily noisy. The stack trace is already part of the test report